joplin/base/migrations/0003_auto_20171128_1944.py

```diff
@@ -0,0 +1,130 @@
+            ],
+            options={
+                'abstract': False,
+            },
+        ),
+        migrations.CreateModel(
+            name='Department',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('name', models.CharField(max_length=255)),
+                ('address', models.CharField(max_length=255)),
+                ('hours', models.CharField(max_length=255)),
+                ('phone_number', models.CharField(max_length=255)),
+                ('email', models.EmailField(max_length=254)),
+                ('description', models.TextField()),
+            ],
+            options={
+                'abstract': False,
+            },
+        ),
+        migrations.CreateModel(
+            name='Director',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('name', models.CharField(max_length=255)),
+                ('photo', models.CharField(max_length=255)),
+                ('email', models.EmailField(max_length=254)),
+                ('phone_number', models.CharField(max_length=255)),
+            ],
+            options={
+                'abstract': False,
+            },
+        ),
+        migrations.CreateModel(
+            name='Event',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('title', models.CharField(max_length=255)),
+                ('start_at', models.DateTimeField()),
+                ('end_at', models.DateTimeField()),
+                ('address', models.TextField()),
+                ('fees', models.CharField(max_length=255)),
+            ],
+            options={
+                'abstract': False,
+            },
+        ),
+        migrations.CreateModel(
+            name='Location',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('name', models.CharField(max_length=255)),
+                ('address', models.TextField()),
+                ('hours', models.CharField(max_length=255)),
+            ],
+            options={
+                'abstract': False,
+            },
+        ),
+        migrations.CreateModel(
+            name='ServicePage',
+            fields=[
+                ('page_ptr', models.OneToOneField(auto_created=True, on_delete=django.db.models.deletion.CASCADE, parent_link=True, primary_key=True, serialize=False, to='wagtailcore.Page')),
+                ('created_at', models.DateTimeField(auto_now_add=True)),
+                ('updated_at', models.DateTimeField(auto_now=True)),
+                ('content', wagtail.wagtailcore.fields.RichTextField(help_text='Write out the steps a resident needs to take to use the service')),
```

Comment:
It'd be cool to implement this as a [streamfield](https://wagtail.io/features/streamfield/) to avoid having the list as just a chunk of HTML

Reply:
StreamField has a pretty awkward UI, so I think it would bog down authors with the way they'd have to create new list entries or new paragraphs. I like the idea of separating content into lists. Filed an issue to do better than just returning HTML cityofaustin/joplin#15